sdk-base/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPlugin.kt

```diff
@@ -11,6 +11,13 @@ import com.mapbox.maps.plugin.MapPlugin
  * Interface to interact with Camera Animations plugin
  */
 interface CameraAnimationsPlugin : MapPlugin {
+
+  /**
+   * Map camera anchor value.
+   * If equal to NULL then center of map view is treated as anchor.
+   */
+  var anchor: ScreenCoordinate?
```

Comment:
Not sure whether this is the best way to expose the value publicly. As far as I can tell, the SDK does not set `NULL` as an anchor anywhere. Developers cannot set the value to `NULL` either, the animator's constructor values are non-nullable. So this will always be `ScreenCoordinate(0, 0)` beside the initial state. Is that worth introducing a new listener and nullable value type? Wondering whether we could just initialize with `ScreenCoordinate(0, 0)` and it would actually be easier to consume.

Reply:
Discussed locally that `ScreenCoordinate(0.0, 0.0)` is not center but top-left corner so this approach will not work. Will be leaving current approach with some clarifying docs.